.ci/eslint-plugin-zotero-translator/index.js

```diff
@@ -79,21 +104,17 @@ module.exports.rules = {
 		return {
 			Program: function(node) {
 				const filename = path.resolve(context.getFilename());
+				const basename = path.basename(filename);
 				const header = headers[filename];
-			
+
 				if (!header || header.error) return; // picked up by valid-json
-			
+
 				if (!header.parsed.lastUpdated) {
 					context.report(node, 'Header needs lastUpdated field');
 
-				} /* else {
-					// disabled until I figure out something smart -- git doesn't retain file modification dates, and it's too involved to hit the github API all the time
-					const stats = fs.statSync(filename)
-					if (stats.mtime > (new Date(header.parsed.lastUpdated))) {
-						context.report(node, 'lastUpdated field is older than file modification time');
-					}
+				} else if (lastUpdated[basename] && lastUpdated[basename] >= header.parsed.lastUpdated) {
+					context.report(node, `lastUpdated field must be updated to be > ${lastUpdated[basename]}`);
```

Comment:
Is this an error or a warning? Should be the latter. Let's also say `lastUpdated field must be updated to > ${lastUpdated[basename]} to push to clients` to make it clear when it's actually necessary.

Reply:
This code just marks its presence, the `.eslintrc` file determines whether it's an error or a warning (or handled at all). I've added the extra text.

Reply:
Ah, OK. So we should make this a warning in .eslintrc. (Technically this is also checking whether there's a `lastUpdated` at all, which should be an error, but I'm not really concerned about that.)

Reply:
Right. I could split this into two rules easily.

Reply:
I don't think that's necessary. There's a lot that could be missing in the metadata header to break things, but it's just not an issue that comes up (since the header is almost always generated by Scaffold anyway).

Reply:
BTW I am probably an outlier but I've never used Scaffold.